src/FunderMaps.WebApi/Controllers/Application/OrganizationUserAdminController.cs

```diff
@@ -20,33 +20,49 @@ namespace FunderMaps.WebApi.Controllers.Application
     ///     This controller provides organization administration.
     ///     <para>
     ///         For the variant based on the current session see 
-    ///         <see cref="OrganizationUserController"/>.
+    ///         <see cref="FunderMaps.AspNetCore.Controllers.OrganizationUserController"/>.
     ///     </para>
     /// </remarks>
     [Authorize(Policy = "AdministratorPolicy")]
     [Route("admin/organization/{id:guid}/user")]
-    public class OrganizationUserAdminController : BaseApiController
+    public class OrganizationUserAdminController : ControllerBase
     {
         private readonly IMapper _mapper;
-        private readonly OrganizationManager _organizationManager;
+        private readonly IUserRepository _userRepository;
+        private readonly IOrganizationUserRepository _organizationUserRepository;
+        private readonly SignInService _signinService;
 
         /// <summary>
         ///     Create new instance.
         /// </summary>
-        public OrganizationUserAdminController(IMapper mapper, OrganizationManager organizationManager)
+        public OrganizationUserAdminController(
+            IMapper mapper,
+            IUserRepository userRepository,
+            IOrganizationUserRepository organizationUserRepository,
+            SignInService signinService)
         {
             _mapper = mapper ?? throw new ArgumentNullException(nameof(mapper));
-            _organizationManager = organizationManager ?? throw new ArgumentNullException(nameof(organizationManager));
+            _userRepository = userRepository ?? throw new ArgumentNullException(nameof(userRepository));
+            _organizationUserRepository = organizationUserRepository ?? throw new ArgumentNullException(nameof(organizationUserRepository));
+            _signinService = signinService ?? throw new ArgumentNullException(nameof(signinService));
         }
 
+        // POST: api/admin/organization/{id}/user
+        /// <summary>
+        ///     Add user to organization.
+        /// </summary>
         [HttpPost]
-        public async Task<IActionResult> AddUserAsync(Guid id, [FromBody] UserDto input)
+        public async Task<IActionResult> AddUserAsync(Guid id, [FromBody] OrganizationUserDto input)
         {
             // Map.
             var user = _mapper.Map<User>(input);
 
             // Act.
-            user = await _organizationManager.AddUserAsync(id, user);
+            // FUTURE: Do in 1 call.
+            var userId = await _userRepository.AddAsync(user);
+            // var passwordHash = _passwordHasher.HashPassword(plainPassword);
```

Comment:
Shouldn't these comments be removed?

Reply:
Conclusion: we will make a DTO extension of `OrganizationUserDTO` with a password field.